tests/yamitranscode.cpp

```diff
@@ -62,6 +62,7 @@ static void print_help(const char* app)
     printf("   --btl1 <svc-t layer 1 bitrate: kbps > optional\n");
     printf("   --btl2 <svc-t layer 2 bitrate: kbps> optional\n");
     printf("   --btl3 <svc-t layer 3 bitrate: kbps> optional\n");
+    printf("   --enable-simulcast <enable h264 simulcast, priorityid also should be set)> optional\n");
```

Comment:
@chivakker ，we don't support full SVC, we just support svc-t. If temporalLayerNum > 1,  that's is svc-t , we don't need another option for svc-t, temporalLayerNum is enough, and it is very clear.